lib/view_component/storybook/controls/simple_control_config.rb

```diff
@@ -9,16 +9,16 @@ module Controls
       class SimpleControlConfig < ControlConfig
         attr_reader :default_value
 
-        def initialize(default_value, param: nil, name: nil)
-          super(param: param, name: name)
+        def initialize(default_value, param: nil, name: nil, description: nil)
+          super(param: param, name: name, description: description)
           @default_value = default_value
         end
 
         def to_csf_params
           validate!
           {
             args: { param => csf_value },
-            argTypes: { param => { control: csf_control_params, name: name } }
+            argTypes: { param => { control: csf_control_params, name: name, description: description } }
           }
```

Comment:
I wonder if we can avoid adding description in the csf params if it's nil (which just feels noisy). A couple of ways to factor this but perhaps just add `.compcat` on the argTypes value?

Reply:
This would dramatically reduce the number of specs that need to change which is a sign of an improvement

Reply:
Great point — this is done 👍